Add KB-H072 and KB-H073 to docs (#13309)
conan-io/conan-center-index: docs/error_knowledge_base.md

    self.cpp_info.includedirs = []
```

#### **<a name="KB-H072">#KB-H072</a>: "PYLINT EXECUTION"**

> Pylint is executed by default over all `conanfile.py` in Conan Center Index and it should not be skipped. It's an important tool which help us keeping a standard level of acceptance, otherwise, it would be incredible hard to review all recipes keeping the same standard level.

Reviewer: ```suggestion
Pylint is executed by default over all `conanfile.py` files in Conan Center Index and it should not be skipped. It's an important tool which helps us keep a standard level of acceptance. Otherwise, it would be incredibly hard to review all recipes and keep them to the same level of standards.
```

> Pylint is executed by default over all `conanfile.py` files in Conan Center Index and it should not be skipped. It's an important tool which helps us keep a standard level of acceptance. Otherwise, it would be incredibly hard to review all recipes and keep them to the same level of standards.

Reviewer 2: Trying to keep it the same everywhere 🙊 

```suggestion
Pylint is executed by default over all `conanfile.py` files in ConanCenterIndex and it should not be skipped. It's an important tool which helps us keep a standard level of acceptance. Otherwise, it would be incredibly hard to review all recipes and keep them to the same level of standards.
```

#### **<a name="KB-H073">#KB-H073</a>: "TEST V1 PACKAGE FOLDER"**

> The legacy content in test_package should not be removed, instead, rename that folder to test_v1_package and create a new test_folder following the [file structure](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#recipe-files-structure) related to Conan v2 and v1 compatibility. Also, you can obtain good example on [templates package](https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates) about Conan v2 test_package.

Reviewer: ```suggestion
The legacy content in test_package should not be removed. Instead, rename that folder to `test_v1_package` and create a new `test_package` folder following the [file structure](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#recipe-files-structure) related to Conan v2 and v1 compatibility. Also, you can obtain good examples of Conan v2 test package's from the [template packages](https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates) folder.
```

> The legacy content in test_package should not be removed. Instead, rename that folder to `test_v1_package` and create a new `test_package` folder following the [file structure](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#recipe-files-structure) related to Conan v2 and v1 compatibility. Also, you can obtain good examples of Conan v2 test package's from the [template packages](https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates) folder.

Reviewer: ```suggestion
The legacy content in test_package should not be removed. Instead, rename that folder to `test_v1_package` and create a new `test_package` folder following the [file structure](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#recipe-files-structure) related to Conan v2 and v1 compatibility. Also, you can obtain good examples of Conan v2 test packages from the [template packages](https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates) folder.
```
